test/libsolidity/syntaxTests/contractBaseLocation/contract_named_at.sol

```diff
@@ -0,0 +1,2 @@
+contract at layout at 0x1234ABC { }
```

Comment:
Shouldn't this be disallowed? `at` should not appear before `layout at`, and `layout at` must appear only once, no?

Reply:
Oh, now I see that the contract is named `at`, my bad hahaha

Reply:
> `at` should not appear before `layout at`

That's a good test case which I didn't think about. I will include it.